packages/api/controllers/user.js

```diff
@@ -20,4 +20,34 @@ const getUserById = async (req, res) => {
   }
 };
 
-module.exports = { getUserById };
+const create = async (req, res) => {
+  const { email, username } = req.body;
+  if (!email) {
+    return res.status(400).send('Missing email');
+  }
+  if (!username) {
+    return res.status(400).send('Missing username');
+  }
+
+  try {
+    const [user, created] = await db.User.findOrCreate({
+      where: {
+        email,
+      },
+      defaults: {
+        username, email,
+      },
+    });
```

Comment:
Not saying it's wrong, but what led you to use `findOrCreate` here? As opposed to `create` + some exception handling.